nff/utils/constants.py

```diff
@@ -61,6 +61,57 @@
     "_grad": 1.0,
 }
 
+
```

Comment:
Seems like the following are repeats of the above but with additional keys. Can we merge them, i.e., delete the ones on top?

Reply:
Done !!